src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -260,19 +255,16 @@ ov::AnyMap get_default_common_config(const std::shared_ptr<ov::Model>& model) {
     } else {
         config.emplace("NPUW_FUNCALL_FOR_ALL", "YES");
     }
-    enable_npuw_dq_if_allowed(config, model);
     return config;
 }
 
 ov::AnyMap get_default_prefill_config(const std::shared_ptr<ov::Model>& model,
                                       const std::optional<NPUDesc>& npudesc) {
-    // NB: Empty config for devices != NPU
-    if (!npudesc.has_value()) {
-        return { };
-    }
```

Comment:
is it ok?

Reply:
Decided not to fallback into CPU but user NPUW:CPU instead